app/src/main/java/com/alphawallet/app/ui/Erc20DetailActivity.java

```diff
@@ -328,6 +328,9 @@ public void handleClick(String action, int actionId)
         if (actionId == R.string.convert_to_xdai)
         {
             openDapp(C.XDAI_BRIDGE_DAPP);
+        } else if (actionId == R.string.swap_with_quickswap) {
+            String queryPath = "?use=v2&inputCurrency=" + token.getAddress();
```

Comment:
Remember to use "ETH" (verbatim) instead of `0x00...00` if it is the native crypto token according to Uniswap docs https://uniswap.org/docs/v2/interface-integration/custom-interface-linking/